packages/account/src/BiconomySmartAccountV2.ts

```diff
@@ -78,7 +78,42 @@ export class BiconomySmartAccountV2 extends BaseSmartAccount {
   activeValidationModule!: BaseValidationModule;
 
   private constructor(readonly biconomySmartAccountConfig: BiconomySmartAccountV2Config) {
-    super(biconomySmartAccountConfig);
+    super({
+      ...biconomySmartAccountConfig,
+      chain: getChain(biconomySmartAccountConfig.chainId),
+      rpcClient: biconomySmartAccountConfig.rpcUrl || (RPC_PROVIDER_URLS[biconomySmartAccountConfig.chainId] as string),
+      entryPointAddress: biconomySmartAccountConfig.entryPointAddress as Hex,
```

Comment:
Here we can also use the default entry point address if not provided in biconomySmartAccountConfig like we do for factoryAddress